src/Components/v2/FollowArtistPopover/FollowArtistPopoverRow.tsx

```diff
@@ -104,7 +106,7 @@ class FollowArtistPopoverRow extends React.Component<Props, State> {
     const { artist: originalArtist } = this.props
     const { swappedArtist } = this.state
     const artist = swappedArtist || originalArtist
-    const imageUrl = get(artist, a => a.image.cropped.url)
+    const imageUrl = get(artist, a => a!.image!.cropped!.url, "")
```

Comment:
Would be be able to modify the `get` function so as to avoid `!`? Not sure if its the best idea, but one nice characteristic about the get function is being able to throw whatever you want inside of it (including complex logical operations), and assuming there's a default fallback things are fine.

Reply:
Hmm... This is actually the thing [`ts-get` does](https://github.com/RIP21/ts-get/blob/master/src/ts-get/index.ts#L34). There's just [the issue](https://github.com/RIP21/ts-get#known-issues) with the return type. Maybe @alloy can poke it.

Reply:
Yeah I don’t see any conditional for function properties. Nonetheless, the commented on code does include function calls and so should already work, no?